internal/xds/rbac/rbac_engine.go

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package rbac provides service-level and method-level access control for a
+// service.
+package rbac
+
+import (
+	"net"
+
+	v3rbacpb "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v3"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/peer"
+)
+
+// AuthorizationDecision is what will be returned from the RBAC Engine when it
+// is asked to see if an incoming RPC matches a policy.
+type AuthorizationDecision struct {
+	MatchingPolicyName string
+}
+
+// Engine is used for making authorization decisions on incoming RPCs.
```

Comment:
Please get rid of all mentions of authorization decisions. We are not doing that here.